data/abilities/command-and-control/1837b43e-4fff-46b2-a604-a602f7540469.yml

```diff
@@ -0,0 +1,26 @@
+- id: 1837b43e-4fff-46b2-a604-a602f7540469
+  name: Elasticat
+  description: A Blue Python agent that executes elasticsearch queries.
+  tactic: command-and-control
+  technique:
+    attack_id: T1071
+    name: Standard Application Layer Protocol
+  platforms:
+    darwin:
+      sh:
+        command: |
+          server="#{app.contact.http}";
+          curl -s -X POST -H "file:elasticat.py" -H "platform:darwin" $server/file/download > elasticat.py;
```

Comment:
why curl instead of specifying as a payload?

Reply:
This ability is the stub needed to have elasticat show up as an option in the 'Deploy Agents' modal. The intended use for the command is to manually launch the new agent.  We could definitely add a new ability to allow other agents to launch it though.